### ISSUE TITLE
Stretch: dependency issue with libmysqlclient-dev

### ISSUE THREAD

**Issue by alexAubin** [YunoHost-Apps/ihatemoney_ynh#20]
Testing the app install on stretch, it seems that `libmysqlclient-dev` is not available, hence the install is broken 😢

**Comment by JocelynDelalande** [comment]
@alexAubin But YNH does not run on stretch yet, right ?

**Comment by alexAubin** [comment]
Yes and no ;)

But it will someday™ ! Which is why we try to make sure that apps can be installed on stretch 😛 

Edit: okay I think your question was actually about "how do I test this". So in that context, the answer is yes: Yunohost kinda runs on Stretch, though some things are likely to be broken, so we're in the process of testing/patching some parts. But the biggest part is basically the app part.

We have a vagrant box with Stretch + YunoHost preinstalled, or you should be able to manually install it with this script : https://github.com/YunoHost/install_script/blob/stretch2/install_yunohost

**Comment by JocelynDelalande** [comment]
Thanks :-).

Happy to hear that people are working on getting ynh to work on stretch :-)

**Comment by JocelynDelalande** [comment]
@alexAubin I dropped the dependency to libmysqlclient-dev since I switched to a new mysql lib (see https://github.com/YunoHost-Apps/ihatemoney_ynh/commit/efc35dd01dcf4692147d103852dc443f1cf9b06e ), that do not require libmysqlclient-dev. So, as a side effect, this dependency issue is no longer, and I close your ticket :-).

Feel free to test it on Stretch though, maybe it will crash later in install process :-p.
